theano/tensor/basic.py

```diff
@@ -3139,7 +3141,9 @@ def var(input, axis=None, keepdims=False):
     centered_input = input - mean_input
 
     # return the mean sqr
-    v = mean((centered_input ** 2), axis, keepdims=keepdims)
+    #divisor = theano.tensor.cast(prod(input.shape[axis]) - ddof, theano.config.floatX)
```

Comment:
check if ddof is True or False. In that case, raise an error and tell that we changed the parameter order. This will help prevent user error due to the parameter order change.